start.sh

```
@@ -0,0 +1,55 @@
+  ./start.sh --debug backup
+  eval "$(./start.sh env)"
+EOF
+}
+
+setup_env() {
+    if [[ ! -d "${VENV_DIR}" ]]; then
+        "${PYTHON_BIN}" -m venv "${VENV_DIR}" >&2
+    fi
+    "${VENV_DIR}/bin/python" -m pip install --upgrade pip >&2
+    "${VENV_DIR}/bin/python" -m pip install -r requirements.txt >&2
+    source "${VENV_DIR}/bin/activate"
```

Shellcheck (0.11.0)

[info] Not following: ./bin/activate was not specified as input (see shellcheck -x).

(SC1091)